apisix/stream/router/ip_port.lua

```diff
@@ -71,6 +82,20 @@ do
             end
 
             local route = item.value
+
+            if not item.value.remote_addr_matcher and validate_address(item.value.remote_addr) then
```

Comment:
Please add the check to https://github.com/apache/apisix/blob/df0db95ce8f8a5a977481539b6af7e056d295680/apisix/stream/router/ip_port.lua#L185